sway/input/text_input.c

```diff
@@ -267,31 +267,37 @@ static void relay_handle_text_input(struct wl_listener *listener,
 	sway_text_input_create(relay, wlr_text_input);
 }
 
+static struct sway_layer_surface *loop_layer_surface(
+		struct wl_list children,
+		struct wlr_layer_surface_v1 *layer)
```

Comment:
nit: no newline here (we place opening curly brackets on the same line as the parameter list)

Reply:
To resolve this, use the following style. Note how the `{` is _not_ on a separate line.

```c
static struct sway_layer_surface *loop_layer_surface(struct wl_list children,
		struct wlr_layer_surface_v1 *layer) {
```